docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -346,10 +356,14 @@ class ComponentExample extends React.PureComponent<IComponentExampleProps, IComp
   private getDisplayName = () => this.props.examplePath.split('/')[1]
 
   private renderWithProvider(ExampleComponent) {
-    const { showRtl, theme } = this.state
+    const { showRtl, componentVariables } = this.state
+    const { themeName } = this.state
```

Comment:
nit: these two lines could be merged into one

Reply:
I commented on previous comment.